### PR TITLE
More troubleshooting tips for Aggregator.

### PR DESCRIPTION
**Pull request by thomasvn**
## Related Issue

N/A

## Proposed Changes

- Remove instructions about modifying `DB_BUCKET_REFRESH_INTERVAL` as that is now outdated
- Include instructions on query and reading `/model/debug/orchestrator`

### CODE REVIEW

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md [hunk 1]
```diff
@@ -100,12 +100,13 @@ kubecostAggregator:
...
+  # default of "no limit" is appropriate when first establishing a baseline of
+  # resource usage required. It is eventually recommended to set these values
+  # such that dbMemoryLimit + dbWriteMemoryLimit < the total memory available
+  # to the aggregator pod.
   # default: 0GB is no limit
   dbMemoryLimit: 0GB
-
-  # Memory limit applied to write database connections.
-  # default: 0GB is no limit
   dbWriteMemoryLimit: 0GB
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/aggregator.md:110]: @mbolt35 This reasonable?

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md [hunk 2]
```diff
@@ -184,6 +187,19 @@ data to be ingested.
...
+```txt
+https://kubecost.myorganization.com/model/debug/orchestrator
+```
+
+This is a common endpoint for debugging the state of the Aggregator. It returns a JSON response with details such as:
+
+* What is Aggregator's current state? If ingesting, it is downloading & processing ETL files into the DB. If deriving it is pre-computing commonly used queries into saved tables.
+* What is the ingestion progress for each of the data types? (e.g. asset, allocation, cloud cost, etc.)
+* How fresh is my read database? An epoch timestamp can be found in the `readDBPath`.
+* How frequently is my newly ingested data being promoted into the read database? Reference `currentBucketRefreshInterval`.
```
Comment (thomasvn) [inline install-and-configure/install/multi-cluster/federated-etl/aggregator.md:201]: @jessegoodier Thoughts? Lmk if I'm missing anything.

Comment (jessegoodier) [reply install-and-configure/install/multi-cluster/federated-etl/aggregator.md:201]: nice work. I can't think of anything else to add at the moment.

**Review by jessegoodier — APPROVED** [review]
thank you!

**Review by srpomeroy — APPROVED** [review]

File: install-and-configure/install/multi-cluster/federated-etl/aggregator.md (hunk 1 above)
Comment (mbolt35) [reply install-and-configure/install/multi-cluster/federated-etl/aggregator.md:110]: Yeah, I think that's fine!

**Pull request merged**